samples/cblas/run.sh

```diff
@@ -1,3 +1,3 @@
-time java --enable-native-access=ALL-UNNAMED --enable-preview --source=22 \
+time java --enable-native-access=ALL-UNNAMED \
```

Comment:
why the `time` in front?

Reply:
That was pre-existing code. Most likely we had it to have simple measure of execution speed of numeric libraries in the past. I can remove it if desired.